### PR TITLE
docs: add `Getting Started` section in README

### LINKED ISSUE #142: [docs] Simple example in main README?
I noticed that after changes in #139 the main `README.md` does not contain a simple workflow example any more.
I believe that this file is also used for the documentation on the [GHA marketplace](https://github.com/marketplace/actions/issue-metrics)?

When trying out a new GitHub Action, I tend to copy and paste a simple example from the documentation, and then adapt it to my purposes from there.

To do that, I find it unhandy if such a workflow example isn't too many clicks away.

I would therefore suggest to leave at least a basic example in the main `README.md`.

Not sure which example workflow would be the most simple to use? Maybe this one?
https://github.com/github/issue-metrics/blob/main/docs/example-workflows.md#fixed-time-example

### PR DESCRIPTION
**Pull request by Okabe-Junya**
close #142 

## Proposed Changes

- add `Getting Started` section in README

## Readiness Checklist

Is the term `Quick Start` really appropriate? There are similar expressions like:

- `Getting Started`
- `Usage`
- `Tutorial`
- `Simple Example/Basic Example`
- ...

### Author/Contributor
- [x] If documentation is needed for this change, has that been included in this pull request
- [x] run `make lint` and fix any issues that you have introduced
- [x] run `make test` and ensure you have test coverage for the lines you are introducing

### Reviewer
- [ ] Label as either `bug`, `documentation`, `enhancement`, `infrastructure`, or `breaking`

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
This PR only involves updates to the documentation, so it won't fail any linting or tests.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,57 @@ Feel free to inquire about its usage by creating an issue in this repository.
 To find syntax for search queries, check out the [documentation on searching issues and pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests)
 or the [documentation on searching discussions](https://docs.github.com/en/search-github/searching-on-github/searching-discussions).
 
+## Quick Start
```
Comment (zkoppert) [inline README.md:25]:
```suggestion
## Getting Started
```

Comment (zkoppert) [reply README.md:25]: I like your suggestion from the PR description of using the term "Getting Started".

**Comment by Okabe-Junya** [comment]
Thank you for the review!!

I approved your suggestion commit, and I've also updated the PR title and the "Proposed Changes" section.

**Review by zkoppert — APPROVED** [review]
Thanks for adding this example! Hopefully this will help speed up the process of getting started for new users!

**Pull request merged**